examples/tensorize_vllm_model.py

```diff
@@ -0,0 +1,16 @@
+from vllm import LLM
+from tensorizer import TensorSerializer
+
+model_ref = "EleutherAI/gpt-j-6B"
+# For less intensive requirements, swap above with the line below:
+# model_ref = "EleutherAI/gpt-neo-125M"
+model_name = model_ref.split("/")[-1]
+# Change this to your S3 bucket.
+s3_bucket = "bucket"
```

Comment:
recommend using the `tensorized` bucked here and dropping the `model_name = model_ref.split("/")[-1]` bit so the example works unmodified out of the box

Reply:
This is serializing though -- no one should be writing to `tensorized`.

Reply:
I'm going to refactor this though